README.md

```diff
@@ -24,6 +24,7 @@ $keycloak = new \Fschmtt\Keycloak\Keycloak(
     baseUrl: 'http://keycloak:8080',
     username: 'admin',
     password: 'admin'
+    version: '20.0.1'
```

Comment:
The version is not required, instead a "preflight" request is made before the first resource is accessed to fetch the version.